packages/babel-plugin-proposal-private-methods/test/fixtures/static-accessors-loose/access-before-declaration/exec.js

```diff
@@ -0,0 +1,29 @@
+expect(() => {
+  class C {
+    static #_ = new C;
+    static get #p() { return C };
+    constructor() {
+      C.#p;
+    }
+  }
+}).toThrow(/attempted to use private field on non-instance/);
```

Comment:
This is exactly #12905, the Babel behaviour on static accessors are not aligned with V8. I will try to tackle in a separate PR. Before that it is just a test case parallel to properties.